serverless/serverless-HA.adoc

```diff
@@ -0,0 +1,59 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-HA"]
+= High availability (HA) using {ServerlessProductName}
+:context: serverless-HA
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Active/passive HA is a standard production attribute of Kubernetes APIs.
+For controllers, active/passive HA helps to ensure that APIs stay operational if disruptions to the controller instances that service those APIs occur.
+
+Running controllers as HA components is necessary to ensure that if an active controller crashes or is deleted, there is another controller instance ready to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+
+[id="leader-election_{context}"]
+== Leader election
+
+Active/passive HA in {ServerlessProductName} is achieved by using leader election.
+When a controller is scaled horizontally, the instances of that controller compete to use a shared resource.
+The instance of the controller that has access to the shared resource at any given time is referred to as the leader.
+
+When using a leader election HA pattern, instances of the controller are already scheduled and running inside the cluster before they are required.
+This is more reliable than using ReplicaSets to schedule controller pods, since if a cluster becomes saturated, a ReplicaSet will not be able to schedule a new controller pod to run.
+
+[id="enable-HA_{context}"]
+== Enabling HA for {ServerlessProductName}
+
+To enable active/passive HA for {ServerlessProductName}, you must enable leader election.
+You can do this by modifying the leader election configuration, and then scaling up the controller replica count.
+
+.Prerequisites
+* This guide requires a current Knative Serving installation. For more information, see the documentation on xref:../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[Installing OpenShift Serverless].
+
+.Procedure
+. Enable leader election for the control plane controllers by entering the following command:
```

Comment:
This isn't quite accurate for us, just create a KnativeServing and we default it for you

Reply:
Removing this, thank you!